packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -131,13 +152,24 @@ export function PanelChrome({
 
   const testid = title ? selectors.components.Panels.Panel.title(title) : 'Panel';
 
+  const collapsibleHeader = (
+    <button type="button" className={styles.clearButtonStyles} onClick={toggleOpen}>
```

Comment:
this is lacking some accessibility requirements:

- when collapsed, the button should have `aria-expanded="false"`
- when not collapsed the button should have `aria-expanded="true"` and `aria-controls="content-id"` where `content-id` is the html id of the element being collapsed, in this case the div at line 257.

you can generate an unique id using `useId` from react.
note that in case the panel is collapsed, `aria-controls` should not be added to the button, as the element with that id won't be rendered.